eessi_bot_job_manager.py

```diff
@@ -291,6 +299,57 @@ def process_new_job(self, new_job):
 
         return True
 
+    def process_running_jobs(self, running_job):
+
+        gh = github.get_instance()
+
+        # set some variables for accessing work dir of job
+        job_dir = os.path.join(self.submitted_jobs_dir, running_job["jobid"])
+
+        # TODO create function for obtaining values from metadata file
+        #        might be based on allowing multiple configuration files
+        #        in tools/config.py
+        metadata_file = "_bot_job%s.metadata" % running_job["jobid"]
+        job_metadata_path = os.path.join(job_dir, metadata_file)
+
+        # check if metadata file exist
+        metadata_pr = self.read_job_pr_metadata(job_metadata_path)
+        if metadata_pr is None:
+            raise Exception("Unable to find metadata file")
+
+        # get repo name
+        repo_name = metadata_pr.get("repo", "")
+        # get pr number
+        pr_number = metadata_pr.get("pr_number", None)
+
+        repo = gh.get_repo(repo_name)
+        pullrequest = repo.get_pull(int(pr_number))
+
+        # determine comment to be updated
+        if "comment_id" not in running_job:
+            running_job_cmnt = get_submitted_job_comment(pullrequest, running_job['jobid'])
+
+            if running_job_cmnt:
+                log(
+                    "process_running_job(): found comment with id %s"
+                    % running_job_cmnt.id,
+                    self.logfile,
+                )
+                running_job["comment_id"] = running_job_cmnt.id
+
+        if "comment_id" in running_job:
+            dt = datetime.now(timezone.utc)
+            update = "\n|%s|" % dt.strftime("%b %d %X %Z %Y")
+            update += "Running|Job id %s is in running state" % running_job['jobid']
+            update_comment(running_job["comment_id"], pullrequest, update)
```

Comment:
Check whether this update has already been included in the comment body, to avoid adding it again:

```python
running_msg = "Job %s is running" % running_job['jobid']
if running_msg in running_job["comment_body"]:
    log("Not updating comment, '%s' already found" % running_msg)
else:
    update = "\n|%s|" % dt.strftime("%b %d %X %Z %Y")
    update += "Running|" + running_msg
    update_comment(running_job["comment_id"], pullrequest, update)